crates/encoding/src/path.rs

```diff
@@ -207,8 +207,8 @@ pub struct SegmentCount {
 #[derive(Clone, Copy, Debug, Zeroable, Pod, Default)]
 #[repr(C)]
 pub struct PathSegment {
-    pub origin: [f32; 2],
-    pub delta: [f32; 2],
+    pub point0: [f32; 2],
```

Comment:
I would repeat the comment here about `point0` and `point1` being relative to tile origin.